shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -515,6 +541,9 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
     // TODO(tvolkert): provide meaningful string (e.g. "page 2 of 5")
     UIAccessibilityPostNotification(UIAccessibilityPageScrolledNotification, @"");
   }
+  if (routeChanged) {
```

Comment:
My understanding from the docs is that we should either send a UIAccessibilityLayoutChangedNotification or UIAccessibilityScreenChangedNotification, but not both? UIAccessibilityScreenChangedNotification is the most specific one, if that applies send that. If not, check if UIAccessibilityLayoutChangedNotification should be send.

Reply:
Done